docs/docs/learn/shards/.pages

```diff
@@ -1,6 +1,4 @@
 nav:
   - index.md
-  - what-is-shards.md
-  - coding-with-shards.md
-  - the-flow.md
-  - working-with-data.md
+  - primer
+  - tutorials
```

Comment:
If we decide to move the tutorials there, we have to also modify the CI job to have them copied at the right place.

Reply:
Yes we'll have to shift the tutorials there since our learn pages will be grouped up by projects. I've renamed the primer folder from "shards" to "primer" too since the parent folder would be "shards" too
.